test/cri-containerd/createcontainer_test.go

```diff
@@ -1152,6 +1153,47 @@ func Test_CreateContainer_Mount_ReadOnlyDir_WCOW(t *testing.T) {
 	runCreateContainerTest(t, wcowHypervisorRuntimeHandler, request)
 }
 
+func Test_CreateContainer_Mount_EmptyDir_WCOW(t *testing.T) {
```

Comment:
I know other tests are named like this, but I would still prefer if tests that only run the hypervisor version of a code path still have "WCOW_Hypervisor" in the name. Thoughts? 

This is a nit though, feel free to say that you won't do this now.

Reply:
I don't have strong feelings, and probably won't change it in this PR, but if you want to make a PR to update a bunch of test names, that would be fine. :)